pkg/utils/string.go

```diff
@@ -14,3 +14,13 @@ func Pluralize(one, many string, count int) string {
 func ToString(val interface{}) string {
 	return fmt.Sprintf("%v", val)
 }
+
+func TruncString(s string, c int) string {
```

Comment:
it might be nice to move these to the polly lib at some point, but not crucial

Reply:
I was actually wanting something more advanced than a simple truncate but couldn't find something better.